db/bus_test.go

```diff
@@ -0,0 +1,597 @@
+                    Hidden: false,
+                    LastSeenAt: 2,
+                    LastCheckedAt: 3,
+                    LastError: "nil",
+                    Status: "good",
+                }, "*")
+            })
+
+            It("should receive a create-object message bus event for Agent, eventually", func(done Done) {
+                /* this is executed in a goroutine */
+                var a Agent
+
+                /* Create agent object bus event*/
+                receive(<-events, "create-object", "agent", &a)
+                Ω(a.UUID).Should(Equal("foo"))
+                Ω(a.Name).Should(Equal("agent"))
+                Ω(a.Address).Should(Equal("addr"))
+                Ω(a.Version).Should(Equal("v1.0"))
+                Ω(a.Hidden).Should(Equal(false))
+                Ω(a.LastSeenAt).Should(Equal(int64(2)))
+                Ω(a.LastCheckedAt).Should(Equal(int64(3)))
+                Ω(a.LastError).Should(Equal("nil"))
+                Ω(a.Status).Should(Equal("good"))
+
+                close(done)
+            }, 2 /* timeout (in seconds) */)
+        })
+
+        Context("when sending an updateObject message for Agent", func() {
+            BeforeEach(func() {
+                /* Update Agent object*/
+                db.sendUpdateObjectEvent(Agent{
+                    UUID:     "foo",
+                    Name:     "agent",
+                    Address:  "addr",
+                    Version:  "v1.1",
+                    Hidden: true,
+                    LastSeenAt: 3,
+                    LastCheckedAt: 4,
+                    LastError: "nil",
+                    Status: "bad",
+                }, "*")
+            })
+
+            It("should receive a update-object message bus event for Agent, eventually", func(done Done) {
+                /* this is executed in a goroutine */
+                var a Agent
+
+                /* Update Agent object message bus event*/
+                receive(<-events, "update-object", "agent", &a)
+                Ω(a.UUID).Should(Equal("foo"))
+                Ω(a.Name).Should(Equal("agent"))
+                Ω(a.Address).Should(Equal("addr"))
+                Ω(a.Version).Should(Equal("v1.1"))
+                Ω(a.Hidden).Should(Equal(true))
+                Ω(a.LastSeenAt).Should(Equal(int64(3)))
+                Ω(a.LastCheckedAt).Should(Equal(int64(4)))
+                Ω(a.LastError).Should(Equal("nil"))
+                Ω(a.Status).Should(Equal("bad"))
+
+                close(done)
+            }, 2 /* timeout (in seconds) */)
+        })
+
+        /* Message bus tests for Job */
+        Context("when sending a createObject message for Job", func() {
+            BeforeEach(func() {
+                /* Create Job object */
+                db.sendCreateObjectEvent(Job{
+                    UUID:     "foo",
+                    Name:     "daily",
+                    Summary:  "A Daily Backup",
+                    KeepN:    2,
+                    KeepDays: 2,
```

Comment:
I would set different values here, to make sure we aren't confusing KeepN / KeepDays